tests/framework/extensions/kubectl/template.go

```diff
@@ -0,0 +1,143 @@
+package kubectl
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeconfig"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rancher/rancher/tests/framework/clients/dynamic"
+	ext_unstructured "github.com/rancher/rancher/tests/framework/extensions/unstructured"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/watch"
+	k8Scheme "k8s.io/client-go/kubernetes/scheme"
+	restclient "k8s.io/client-go/rest"
+)
+
+const (
+	// rancherShellSettingID is the setting ID that used to grab rancher/shell image
+	rancherShellSettingID = "shell-image"
+	Namespace             = "kube-system"
+	JobName               = "kubectl"
+)
+
+var (
+	importTimeout = int64(60 * 1)
+	group         int64
+	user          int64
+	Job           = &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "",
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "",
+				},
+				Spec: corev1.PodSpec{
+					RestartPolicy:      "Never",
+					ServiceAccountName: "",
+					Containers:         []corev1.Container{},
+					Volumes: []corev1.Volume{
+						{
+							Name: "config",
+							VolumeSource: corev1.VolumeSource{
+								EmptyDir: &corev1.EmptyDirVolumeSource{},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
```

Comment:
I don't think we need this; it's probably a missed thing. `NewJobTemplate` implemented.

Reply:
You're right. I forgot to update